libs/wingc/src/parser.rs

```diff
@@ -199,14 +199,29 @@ impl Parser<'_> {
 			"block" => StmtKind::Scope(self.build_scope(statement_node)),
 			"if_statement" => {
 				let if_block = self.build_scope(&statement_node.child_by_field_name("block").unwrap());
+
+				let mut elif_vec: Vec<ElifBlock> = Vec::new();
```

Comment:
Lets try to get rid of explicit type annotations, It's probably not needed here.
```suggestion
				let mut elif_vec = vec![];
```